paimon-core/src/main/java/org/apache/paimon/mergetree/compact/PartialUpdateMergeFunction.java

```diff
@@ -214,7 +214,10 @@ private Factory(Options options, RowType rowType) {
                                                             "Field %s is defined repeatedly by multiple groups: %s",
                                                             fieldNames.get(field), k));
                                         }
-                                        fieldSequences.put(field, sequenceGen);
+
+                                        if (field >= 0) {
```

Comment:
I think better way is throw an exception with meaningful message?

Reply:
If we  just define a sequenceGroup with  fields which not in table schema, It don’t report any Exception, just ignore.Because it don't affect write .So I think sequenceGroup is repeat defined with  fields which not in table schema should't report Exception,it can alse write.